provider/cmd/pulumi-resource-gitlab/schema.json

```diff
@@ -14,21 +14,22 @@
     },
     "language": {
         "csharp": {
-            "compatibility": "tfbridge20",
+            "packageReferences": {
```

Comment:
Switched from maps to structs in the bridge and unfortunately this means the order of the fields has changed in the generated JSON. The new order is forward stable but it's different. The semantic should be the same.